packages/compiler-sfc/__tests__/compileScript.spec.ts

```diff
@@ -240,14 +240,22 @@ describe('SFC compile <script setup>', () => {
       const { content } = compile(
         `
       <script setup>
-      import { ref } from 'vue'
-      let foo = $ref(1)
+      import { useCssVars, ref } from 'vue'
+      const msg = ref()
       </script>
-      `,
-        { reactivityTransform: true }
+      
+      <style>
+      .foo {
+        color: v-bind(msg)
```

Comment:
Why did we change the original test case here? 🤔

Reply:
`$ref` will import `ref` originally, but now `$ref` is longer exists. So I imported `useCssVars` and used `v-bind` in <style> block.

Reply:
uh, got your point. (I missed the test name)